src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -27,6 +28,11 @@ import { presentAgentId, presentHolofuelAmount, sliceHash, useLoadingFirstTime,
 import { caribbeanGreen } from 'utils/colors'
 import { OFFER_REQUEST_PATH } from 'holofuel/utils/urls'
 import { TYPE, STATUS, DIRECTION, shouldShowTransactionInInbox } from 'models/Transaction'
+import { userNotification } from 'data-interfaces/HoloFuelDnaInterface'
```

Comment:
we should avoid this pattern. I'm surprised it works. 

Instead we should do something like have a `stale` flag on the transaction object, and catch that in the frontend.

But let's talk about exactly what the problem is that this solves and figure out a more robust way of doing it.